feature-libs/order/core/facade/my-account-v2-order-history.service.ts

```diff
@@ -195,27 +195,7 @@ export class MyAccountV2OrderHistoryService {
     });
   }
 
-  //TODO: CXINT-2896: Remove this method in next major release
-  /**
-   * @deprecated since 2211.20. Use getOrderDetailsV2 instead
-   */
-  getOrderDetails(code: string): Observable<Order> {
-    const loading$ = this.getOrderDetailsState(code).pipe(
-      auditTime(0),
-      tap((state) => {
-        if (!(state.loading || state.success || state.error)) {
-          this.loadOrderDetails(code);
-        }
-      })
-    );
-    return using(
-      () => loading$.subscribe(),
-      () => this.getOrderDetailsValue(code)
-    );
-  }
-
-  //TODO: CXINT-2896: Rename this method to `getOrderDetails` in next major release
-  getOrderDetailsV2(code: string): Observable<Order | undefined> {
+  getOrderDetails(code: string): Observable<Order | undefined> {
```

Comment:
`major:` Since the `getOrderDetails` method was deprecated with the information to use `getOrderDetailsV2` instead, we should keep the one with V2 suffix to be in line with the deprecation message (and to be fair to our customers).